js_modules/dagster-ui/packages/ui-core/src/graphql/schema.graphql

```diff
@@ -2415,7 +2415,7 @@ type InstigationState {
   typeSpecificData: InstigationTypeSpecificData
   runs(limit: Int): [Run!]!
   runsCount: Int!
-  tick(timestamp: Float, tickId: Int): InstigationTick
+  tick(timestamp: Float, tickId: Int): InstigationTick!
```

Comment:
I would expect `timestamp` to have been removed by the schema regeneration. Does something else need to be done to remove this in Python?

Reply:
Yeah, missed a spot. Thanks.